Resources/Prototypes/Entities/Objects/Specific/Medical/disease.yml

```diff
@@ -19,6 +19,27 @@
   # - Virology (when it's back)
     - Botany
 
+- type: entity
+  parent: BaseAmmoProvider # trust
+  id: BoxMouthSwab
+  name: sterile swab dispenser
+  description: Dispenses 30 sterile swabs, extremely useful for botany.
+  components:
+  - type: Sprite
+    layers:
+    - state: boxwide
+    - state: swab
+  - type: BallisticAmmoProvider
```

Comment:
This feels wrong, but I don't know enough about the gun system to be able to tell you why. I hope this can't be "cycled" or put into guns?

Reply:
cant be put into guns like shotgun shell dispensers, hence trust

the only way to remove an item from a BAP is cycling

Reply:
This is gonna be the sussiest code I've merged this year.